source/common/grpc/status.cc

```diff
@@ -1,5 +1,8 @@
 #include "common/grpc/status.h"
 
+#include <string>
```

Comment:
nit: not used

Reply:
The `#include <unordered_map>` is also not used, I'll remove that too.

Reply:
Removed in c64da70.